api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -22,7 +22,8 @@ package v1beta1
 import (
 	networkingv1beta1 "github.com/banzaicloud/istio-client-go/pkg/networking/v1beta1"
 	metav1 "github.com/cert-manager/cert-manager/pkg/apis/meta/v1"
-	"k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Could you double check if this change is needed, please?

Reply:
@azun can you check this please?

Reply:
This is generated code. I'm not sure why it's adding `k8s.io/apimachinery/pkg/apis/meta/v1` which forces the alias for corev1